scripts/demo/2015-10-dev2.sh

```diff
@@ -255,7 +255,7 @@ function run_command()
 	"mkfs")
 		ssh -n $CONFD_NODE systemctl start motr-mkfs@confd
 		ssh -n $CONFD_NODE systemctl start motr-server-confd
-		i=0; for n in $SERVER_LIST; do ssh -n $n systemctl start motr-mkfs@ios$i & i=$(($i + 1)); done
+		i=0; for n in $SERVER_LIST; do ssh -n "$n" systemctl start motr-mkfs@ios$i & i=$(($i + 1)); done
```

Comment:
Please make sure if the following variable need the same change:
$SERVER_LIST
$CONFD_NODE

Reply:
done